src/api/app/views/webui/request/_build_results_filters.haml

```diff
@@ -0,0 +1,28 @@
+.row.mb-3
+  .col-md-12
+    = form_tag(filter_url, method: :get) do
+      = render partial: 'webui/shared/checkboxes_dropdown', locals: { id: 'project-monitor-package-dropdown',
```

Comment:
`checkboxes_dropdown` is too generic... Maybe something more specific like `build_results_filter`?